nbs/60_medical.imaging.ipynb

```diff
@@ -80,6 +80,56 @@
     "_all_ = ['DcmDataset', 'DcmTag', 'DcmMultiValue', 'dcmread', 'get_dicom_files']"
```

Comment:
<p>There's no reason to use two lines here</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/fastai/fastai/pull/2732/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>

Reply:
<p>combined into 1 line and got rid of the extra stuff</p>